prusti-tests/tests/cargo_verify/test_map/src/main.rs

```diff
@@ -0,0 +1,9 @@
+//
+extern crate prusti_contracts;
+use prusti_contracts::*;
```

Comment:
Could you please move these into the regular `https://github.com/viperproject/prusti-dev/tree/master/prusti-tests/tests/verify_overflow/fail/ghost`?

Reply:
After you removed the unnecessary files, could you please also squash the commits? I will then merge your PR.